bazel/grpc_python_deps.bzl

```diff
@@ -4,21 +4,12 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@com_github_grpc_grpc//third_party/py:python_configure.bzl", "python_configure")
 
 def grpc_python_deps():
-    native.bind(
```

Comment:
Does this constitute a breaking change for those who consume us?

Reply:
I see that we're forced to do this [because protobuf already made this breaking change](https://github.com/protocolbuffers/protobuf/pull/6310/files). :/

@nicolasnoble Thoughts on *yet another* Bazel breaking change. As far as I can tell, the options are:
 - accept the breaking change and force *all* of our downstream users to do the same
 - get protobuf to roll back the change and potentially break users who have already accommodated it

Regardless, it's pretty clear that protobuf needs something like a Bazel distribtest.

Reply:
@nicolasnoble and I talked offline. I agree that moving away from `bind` is the right option. But we can't just break backwards compatibility whenever we like. We'd like to batch up several backwards incompatible changes and synchronize them with our addition of official support for Bazel 1.0. We'll be discussing this in a meeting early next week. Until then, we'll need to put this change on hold.